Ocer/SarapanViewController.swift

```diff
@@ -8,22 +8,292 @@
 import UIKit
 
 class SarapanViewController: UIViewController {
-
+    var center: CGPoint = CGPoint.zero
+    var dropZone : CGRect = CGRect(x: 0, y: 0, width: 0, height: 0)
+    func setBackground() {
+        let background = UIImageView(image: UIImage(named: "backgroundMakan"))
```

Comment:
make this as reusable function which can be use from any view controller instead